qiita_pet/templates/study_description.html

```diff
@@ -148,11 +148,10 @@
         break;
     }
   }
-
   if (sff.length === 0 && barcodes.length === 0 && forward.length === 0 && fasta.length === 0) {
     bootstrapAlert("You need to select at least: one barcode and one forward file, one sff file or one fasta file.");
     return [false];
-  } else if (sff.length === 0 && barcodes.length != forward.length) {
+  } else if (sff.length === 0 && barcodes.length != forward.length && filetype !== 5 && filetype != "per_sample_FASTQ") {
```

Comment:
Just that there are the same number of fwd and reverse files, and that test is in place.